src/handlers/motions/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as handleMotionCreated } from './motionCreated';
 export { default as handleMotionStaked } from './motionStaked';
+export { default as handleMotionFinalized } from './motionFinalized/motionFinalized';
```

Comment:
Since you've got an index file in `motionFinalized` you probably don't need to double `motionFinalized` in the path?